source/developer/interactive-messages.rst

```diff
@@ -200,6 +205,9 @@ Parameters
 
 Below is a brief description of each parameter to help you customize the interactive message button and menu in Mattermost. For more information on message attachments, `see our documentation <https://docs.mattermost.com/developer/message-attachments.html>`__.
 
+ID
+  A per post unique identifier.
```

Comment:
```suggestion
  A unique identifier required for each post.
```

Reply:
@justinegeffen That is actually not what I meant. The ID is unique per post. It's not required for every post action. How can I make this more clear?

Reply:
I think we can go with what you've got. No need to over-complicate it - the wording does make sense.